Scala/Scala.sublime-syntax

```diff
@@ -249,16 +304,26 @@ contexts:
                 3: variable.import.renamed-to.scala
             - match: '([^\s.,}]+)'
               scope: variable.import.scala
+
   inheritance:
     - match: '(extends|with)\s+([^\s\{\(\[\]]+)'
       captures:
         1: keyword.declaration.scala
         2: entity.other.inherited-class.scala
+    - match: '(extends|with)\s+\('
```

Comment:
There should, but you'll note that this line is almost a copy/paste of the line above it, which has been in the mode since the dawn of time.  :-)